sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/KeyMgmtBundle.java

```diff
@@ -83,18 +102,27 @@ public KeyMapper createKeyMapper(String keyDefinitionName) {
   // --------------
   // ----- Builder
   // --------------
+  public enum FailStrategy {
+    PASSTHROUGH,
+    FAIL_WITH_EXCEPTION
+  }
+
   public interface InitialBuilder {
     <C extends Configuration> FinalBuilder<C> withKeyMgmtConfigProvider(
         KeyMgmtConfigProvider<C> configProvider);
   }
 
   public interface FinalBuilder<C extends Configuration> {
+
+    FinalBuilder<C> withNoMappingStrategy(FailStrategy strategy);
```

Comment:
* wouldn't `withFailStrategy` be more suitable?
* one or two sentences as documentation would be helpful

Reply:
thanks for the quick review. I considered both comments with the recent push.